webui/react/src/pages/WorkspaceList/WorkspaceCard.tsx

```diff
@@ -44,12 +44,9 @@ const WorkspaceCard: React.FC<Props> = ({ workspace, fetchWorkspaces }: Props) =
           </div>
           <div className={css.info}>
             <div className={css.nameRow}>
-              <Typography.Title
-                className={css.name}
-                ellipsis={{ rows: 1, tooltip: true }}
-                level={5}>
+              <Title size={TypographySize.XS} truncate={{ rows: 1, tooltip: true }}>
                 {workspace.name}
-              </Typography.Title>
+              </Title>
               {workspace.pinned && <Icon name="pin" title="Pinned" />}
             </div>
             <p className={css.projects}>
```

Comment:
the changes here are not looking right to me, in terms of vertical spacing of the text or the horizontal position of the pin icon. Vertical spacing also doesn't look right on latest-main right now, perhaps because of some hew changes that got merged before this determined PR? not sure.

I think if you copy and paste the markup and CSS I posted above, it should more or less match what we had before using the hew components. 

only difference from what I posted should be keeping the `.archivedBadge` class.

Reply:
ok I updated hew so I could get `<Row justifyContent` back and base it on this.

Just pushed an update